themes/soe_basic/src/scss/theme/_events.scss

```diff
@@ -289,3 +289,16 @@
     margin-top: 0;
   }
 }
+
+// Event List paragraph, card view.
+.view.stanford-events.cards {
+  .su-list-unstyled.grid-container-3 {
+    display: flex;
+    justify-content: flex-start;
+    gap: 1em;
+
+    li {
+      flex: 1 0 29;
```

Comment:
@imonroe Can you explain what you want to do here? This isn't valid CSS. Do you want all the cards evenly sized? 
<img width="331" alt="Screenshot 2023-12-13 at 9 58 13 AM" src="https://github.com/SU-SOE/soe_profile/assets/1187335/a6744e11-5b4d-43aa-b05b-02d51e075c2f">

Reply:
I think this is better: `flex: 1 1 min-content;`
It is evenly divided if it is 1, 2 or 3 cards.

Reply:
Ok, I'll try that.  FWIW, that is supposed to mean: 
- flex-grow is ok
- flex-shrink is not ok
- flex-basis is initially 29%.  I did forget the percentage there, which is why it's not valid.

Reply:
Thanks!

Reply:
Your suggestion works great, Jen.  I've made the change.